docs/enrollment-pathways/older-adults.md

```diff
@@ -6,9 +6,9 @@ Currently, the app uses [Login.gov's Identity Assurance Level 2 (IAL2)](https://
 
 ## Demonstration
 
-Here's a GIF showing what the flow looks like, having older adults confirm eligibility via Login.gov and enroll via LittlePay:
+The video walkthough below demonstrates the flow for an older adult who has a Login.gov account to confirm transit benefit eligibility and enroll their bank card for reduced fares via LittlePay:
 
-![Demonstration of the sign-up process for a senior confirming eligibility via Login.gov and enrolling via Littlepay](img/senior-success.gif){ width="350" }
+https://github.com/cal-itp/benefits/assets/6279581/139c77fa-ac75-4189-8b20-61d91051b264
```

Comment:
```suggestion
![Demonstration of the sign-up process for a senior confirming eligibility via Login.gov and enrolling via Littlepay](https://github-production-user-asset-6210df.s3.amazonaws.com/6279581/305587214-139c77fa-ac75-4189-8b20-61d91051b264.mp4?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAVCODYLSA53PQK4ZA%2F20240220%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20240220T214403Z&X-Amz-Expires=300&X-Amz-Signature=8b06c9bc8f6afc89623600f0b6ecebffeaed5509c8442dd4ff48732b49451f50&X-Amz-SignedHeaders=host&actor_id=3673236&key_id=0&repo_id=285501392){ width="350" }
```

Reply:
@machikoyasuda Can you elaborate on what the changes to the URL do? Should I make these changes to the URL going forward when I upload a new version of the video?

Reply:
The URL you first used is not a URL to a video file (like `.mp4`). It is a GitHub.com URL that redirects to a MP4. You need to embed a video file (a file that ends in something like `.gif`, `.mp4`). And by embed, I mean the code has to be like this:

```md
![Alternative text goes here](URL TO GIF/MP4 GOES HERE){width="350"}
```

Instead of this:
```md
link to something that is not a gif
```

Reply:
I see. I simply dragged and dropped the file into the doc. How do I reveal the URL that links directly to the file? (Looks like AWS storage.)

Reply:
Click on this https://github.com/cal-itp/benefits/assets/6279581/139c77fa-ac75-4189-8b20-61d91051b264

Look at the real URL

Reply:
<img width="1407" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/b721c725-e51c-4101-9749-ce7e678c9d73">
actually a totally different URL appears

Reply:
Got it. Thanks!

Reply:
@indexing I think you want to `Commit suggestion` from Machiko's suggestion above before marking this as resolved. I didn't see the extra commit come in that fixes the video URL.